lib/functions/warn_spec.lua

```diff
@@ -2,7 +2,22 @@ local warn = import("./warn")
 
 describe("functions.warn", function()
 	it("should be a function", function()
-		-- Not completely sure how to test this function yet.
 		assert.is_function(warn)
 	end)
+
+	it("should warn", function()
+	   local oldErr = io.stderr
```

Comment:
It looks like this block has mixed indentation between tabs and spaces!

Reply:
Blame my environment, heh. I'll fix it.